application/config/init.php

```diff
@@ -28,7 +28,7 @@
 	'base_url'    => '/',
 	'index_file'  => FALSE,
 	'charset'     => 'utf-8',
-	'errors'      => FALSE,
+	'errors'      => true,
```

Comment:
should be false

Reply:
no no , i think it should be true I accidentally reversed it earlier so this is tot fix it